### DISCUSSION TITLE
--fallback-bc is equal  oklch(var(--bc)/0.2 ?  .btn-ghost.btn-active in chrome 93 , backgroud color is #1f2937 , in edge 119, background color is #d3d4d7

### DISCUSSION THREAD

**Discussion by wenzhili** [saadeghi/daisyui#2580] [Q&A] [answered]
.btn-ghost.btn-active {
    border-color: transparent;
    background-color: var(--fallback-bc,oklch(var(--bc)/0.2))}

**Comment by saadeghi** [comment, marked as answer]
It's not equal. Fallback colors can't support background opacity values.  
Basically fallback colors are already costing a lot in CSS size but we included them anyway just to keep websites working in older browsers. 
There's no plans to increase the compatibility for old browsers in daisyUI 4 and next major version (5) because there are even more new CSS features we have to include in the library.
